keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -612,25 +644,29 @@ to CNP rules.
 #### Sample spec for Story 1: Deny traffic at a cluster level
 
 ```yaml
-apiVersion: netpol.networking.k8s.io/v1alpha1
+apiVersion: policy.networking.k8s.io/v1alpha1
```

Comment:
There was a yaml spec change for this Story in my first commit but seems lost during rebasing?

Reply:
I think I had an overriding change here... Specifically I changed the sample policy to day "deny all ingress traffic to `sensitive-ns`" before It said the inverse I believe